ROADMAP.md

```diff
@@ -23,11 +23,12 @@ In the meanwhile, we are looking for contributions.  An easy place to start is t
 	* Native Binary high-performance format
 
 ### Featurization Improvements
+We already provide text/NLP and image processing functionnalities that will be expanded
 * Text  (*)
-  * Natural language text preprocessing such as tokenization, part-of-speech tagging, and sentence breaking
-  * Pre-trained text models that can be used for extracting of semantic or sentiment features from text
+  * Natural language text preprocessing such as improving tokenization features, adding part-of-speech tagging, and sentence boundary disambiguation
+  * Pre-trained text models (beyond current n-gram text handling) such as pre-trained WordEmbedding that can improve the extraction of semantic or sentiment features from text
```

Comment:
The WordEmbedding transform was added to the external repo in https://github.com/dotnet/machinelearning/pull/545

Reply:
Ok I'll update that then!